core/src/test/java/org/mache/TestEnvironmentPortCheckIgnoreCondition.java

```diff
@@ -0,0 +1,30 @@
+package org.mache;
+
+import com.codeaffine.test.ConditionalIgnoreRule;
+
+public abstract class TestEnvironmentPortCheckIgnoreCondition implements ConditionalIgnoreRule.IgnoreCondition
+{
+    public TestEnvironmentPortCheckIgnoreCondition(int port)
+    {
+        if(new NotRunningInExcelian().isSatisfied()) {
+            if (PortCheck.PortAccessible("localhost", port)) {
+                hostName = "localhost";
+                FOUND_IT = true;
+            }
+        }
+        else {
+            hostName = "10.28.1.140";// internal test environment - we don't check port as we'd like to fail if service is down
+            FOUND_IT = true;
+        }
+    }
+
+    private static String hostName;
+    private static boolean FOUND_IT=false;
```

Comment:
If set by an instance method, this `FOUND_IT` should probably not be static.  The method could perhaps be static also instead?